classes/SnippetManager.php

```diff
@@ -35,13 +35,58 @@ public function listSnippets($theme)
         $themeSnippets = $this->listThemeSnippets($theme);
         $componentSnippets = $this->listComponentSnippets();
 
-        $snippets = array_merge($themeSnippets, $componentSnippets);
+        $this->snippets = array_merge($themeSnippets, $componentSnippets);
+
+        /*
+         * @event pages.snippet.listSnippets
```

Comment:
@tomaszstrojny snippets needs to be plural.
```suggestion
         * @event pages.snippets.listSnippets
```